modules/site-health/audit-full-page-cache/load.php

```diff
@@ -0,0 +1,174 @@
+<?php
+/**
+ * Module Name: Audit Full Page Cache
+ * Description: Adds a check for full page cache in Site Health status.
+ * Experimental: No
+ *
+ * @package performance-lab
+ * @since n.e.x.t
+ */
+
+/**
+ * Require helper functions.
+ */
+require_once __DIR__ . '/helper.php';
+
+/**
+ * Adds tests to site health.
+ *
+ * @since n.e.x.t
+ *
+ * @param array $tests Site Health Tests.
+ * @return array
+ */
+function perflab_afpc_add_full_page_cache_test( $tests ) {
+	$tests['async']['perflab_page_cache'] = array(
+		'label'             => __( 'Page caching', 'performance-lab' ),
+		'test'              => rest_url( 'performance-lab/v1/tests/page-cache' ),
+		'has_rest'          => true,
+		'async_direct_test' => 'perflab_afpc_page_cache_test',
+	);
+	return $tests;
+}
+add_filter( 'site_status_tests', 'perflab_afpc_add_full_page_cache_test' );
+
+/**
+ * Callback for perflab_page_cache test.
+ *
+ * @since n.e.x.t
+ *
+ * @return array
+ */
+function perflab_afpc_page_cache_test() {
+	$description  = '<p>' . __( 'Page caching enhances the speed and performance of your site by saving and serving static pages instead of calling for a page every time a user visits.', 'performance-lab' ) . '</p>';
+	$description .= '<p>' . __( 'Page caching is detected by looking for an active page caching plugin as well as making three requests to the homepage and looking for one or more of the following HTTP client caching response headers:', 'performance-lab' ) . '</p>';
+	$description .= '<code>' . implode( '</code>, <code>', array_keys( perflab_afpc_get_page_cache_headers() ) ) . '.</code>';
+
+	$result = array(
+		'badge'       => array(
+			'label' => __( 'Performance', 'performance-lab' ),
+			'color' => 'green',
+		),
+		'description' => wp_kses_post( $description ),
+		'test'        => 'perflab_page_cache',
+		'status'      => 'good',
+		'label'       => '',
+		'actions'     => sprintf(
+			'<p><a href="%1$s" target="_blank" rel="noopener noreferrer">%2$s <span class="screen-reader-text">%3$s</span><span aria-hidden="true" class="dashicons dashicons-external"></span></a></p>',
+			esc_url( 'https://wordpress.org/support/article/optimization/#Caching' ),
+			__( 'Learn more about page caching', 'performance-lab' ),
+			/* translators: The accessibility text. */
+			__( '(opens in a new tab)', 'performance-lab' )
+		),
+	);
+
+	$page_cache_detail = perflab_afpc_get_page_cache_detail();
+
+	if ( is_wp_error( $page_cache_detail ) ) {
+		$result['badge']['color'] = 'orange';
+		$result['label']          = __( 'Unable to detect the presence of page caching', 'performance-lab' );
+		$result['status']         = 'recommended';
+		$error_info               = sprintf(
+		/* translators: 1 is error message, 2 is error code */
+			__( 'Unable to detect page caching due to possible loopback request problem. Please verify that the loopback request test is passing. Error: %1$s (Code: %2$s)', 'performance-lab' ),
```

Comment:
Indentation is off here:
```suggestion
			/* translators: 1 is error message, 2 is error code */
			__( 'Unable to detect page caching due to possible loopback request problem. Please verify that the loopback request test is passing. Error: %1$s (Code: %2$s)', 'performance-lab' ),
```